src/diffusers/pipelines/cogvideo/pipeline_cogvideox.py

```diff
@@ -577,8 +578,6 @@ def __call__(
         if isinstance(callback_on_step_end, (PipelineCallback, MultiPipelineCallbacks)):
             callback_on_step_end_tensor_inputs = callback_on_step_end.tensor_inputs
 
-        height = height or self.transformer.config.sample_size * self.vae_scale_factor_spatial
-        width = width or self.transformer.config.sample_size * self.vae_scale_factor_spatial
```

Comment:
We don't need it because `height` and `width` are already at their default values.